__tests__/Unit/hooks/usersApi.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { setupServer } from 'msw/node';
+import { useGetUsersQuery, useGetUsersByLinkQuery, useGetUsersByUsernameQuery } from "@/app/services/usersApi";
+import handlers from '../../../__mocks__/handlers';
+import React, { PropsWithChildren } from 'react';
+import { act, renderHook } from '@testing-library/react-hooks';
+import { Provider } from 'react-redux';
+import { store } from "@/app/store";
+
+const server = setupServer(...handlers);
+
+beforeAll(() => {
+  server.listen();
+});
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+function Wrapper({ children }: PropsWithChildren<{}>): JSX.Element {
+  return <Provider store={store()}>{children}</Provider>;
+}
+
+describe('useGetUsersQuery', () => {
+  test('returns users', async () => {
+    const { result, waitForNextUpdate } = renderHook(() => useGetUsersQuery({ size: 2 }), {
+      wrapper: Wrapper,
+    });
+    const initialResponse = result.current;
+    expect(initialResponse.data).toBeUndefined();
+    expect(initialResponse.isLoading).toBe(true);
+
+    await act(() => waitForNextUpdate());
+    const nextResponse = result.current;
+
+    expect(nextResponse.data).not.toBeUndefined();
+    expect(nextResponse.isLoading).toBe(false);
+    expect(nextResponse.isSuccess).toBe(true);
```

Comment:
Where is assert for checking response data contains all users the limit is 2.

Reply:
got it

Reply:
done